### ISSUE TITLE
Better error message when connecting to unconfigured Drupal site

### ISSUE THREAD

**Issue by webchick** [acquia/drupal-create#2]
![Screen Shot 2013-04-03 at 3 09 28 PM](https://f.cloud.github.com/assets/332535/336559/3329f018-9cab-11e2-81ef-d24bdbd694db.png)

Would love some more pointers here, such as a link to where there's information on configuring my site.

**Comment by chrisbrookins** [comment]
The app is designed for end users who create content and likely do not have
any admin access to their site,  The UX design behind this message is that
the end user will then contact their site admin who has all of the
documentation we posted to http://drupal.org on the MASt module that they
configured and will take it from there.  e.g. they will follow the
configuration set up procedure for the module and content types.
- Chris Brookins  :  VP Engineering Acquia

**Comment by webchick** [comment]
Hm. I can see that, I guess. Still, in order for my content authors to be able to use this in the first place I as a site admin need to know how to set it up. :D

What about...

> "This website does not appear to be a Drupal site configured to support this application. Please contact the site
> administrator. If you are the site administrator, <a href="??">read the docs</a>."

**Comment by webchick** [comment]
Or at least, I imagine a lot of other Drupal site developers will do what I did: Stumble onto this project via Twitter or something and go "OH! COOL! I WANT TO TRY THIS!", clone it like they clone any other Github repo, not read the README.md because they never read README.md, and press the play button in Xcode and see this very inviting box for a URL that says "Drupal" at the top, and then type in the URL to their blog and then go "Huh." ;)

However, if I'm an edge case, that's fine to won't fix this. :)

**Comment by weitzman** [comment]
Agreed ... We are working on a new section for README.md which discusses building/configuring the app.

**Comment by lightflows** [comment]
Any update on this - we're trying to configure this app but are having no joy - a step by step setup would be great.
